inc/search-replace-database.php

```diff
@@ -0,0 +1,113 @@
+
+    // Get form input values
+    $search_for = isset($_POST['search_value']) ? $_POST['search_value'] : "$site_url";
+    $replace_with = isset($_POST['replace_value']) ? $_POST['replace_value'] : "";
+    ?>
+
+    <form method="post" action="site-info.php?action=run-search-and-replace&id=<?php echo $blogid; ?>">
+        <hr><h3>Search and Replace</h3>
+        <p>Run a search and replace against keywords in the database, limited to this site.</p>
+
+        <table>
+            <tr>
+                <td>
+                    <?php wp_nonce_field('edit-site'); ?>
+
+                    <input type="hidden" name="id" value="<?php echo esc_attr($blogid); ?>" />
+                    <table class="form-table" role="presentation">
+                        <tr class="form-field">
+                            <th scope="row"><label for="search_value"><?php _e('Search for'); ?></label></th>
+                            <td><input name="search_value" type="text" id="search_value" style="width: 680px"
+                                       value="<?php echo esc_attr($search_for); ?>"></td>
+                        </tr>
+                        <tr class="form-field">
+                            <th scope="row"><label for="replace_with"><?php _e('Replace with'); ?></label></th>
+                            <td><input name="replace_value" type="text" id="replace_value" style="width: 680px"
+                                       value="<?php echo esc_attr($replace_with); ?>"></td>
+                        </tr>
+                        <tr class="form-field">
+                            <th scope="row"><label for="checkbox_field"><?php _e('Dry run'); ?></label></th>
+                            <td><input name="dryrun_value" type="checkbox" id="dryrun_value"
+                                       <?php echo checked($checkbox_field, 1); ?>>
+                                <label for="checkbox_field"></label>
+                            </td>
+                        </tr>
+                    </table>
+                </td>
+                <td></td>
+            </tr>
+        </table>
+
+        <?php
+        // Submit form
+        submit_button('Run search and replace', 'secondary', 'run-search-and-replace', true, "");
+        ?>
+    </form>
+
+    <?php
+        // This styling is needed to surpress the info-php page's button duplicating
+        // on the page.
+    ?>
+    <style>
+        .submit .button {
+            display: none;
+        }
+
+        #run-search-and-replace, #primary-save {
+            display: block;
+        }
+    </style>
+
+    <?php
+    // Set form input values
+    $search_for = isset($_POST['search_value']) ? $_POST['search_value'] : "";
+    $replace_with = isset($_POST['replace_value']) ? $_POST['replace_value'] : "";
+    $dry_run = isset($_POST['dryrun_value']) ? $_POST['dryrun_value'] : "";
+
+    // Set dry run flag
+    $dry_run = ($dry_run === 'on') ? "--dry-run" : "";
+
```

Comment:
super admins should only have access to the is page but to double portect it i would add a [is_super_admin](https://developer.wordpress.org/reference/functions/is_super_admin/) if